src/components/Resources/ResourcePage.js

```diff
@@ -3,46 +3,110 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import axios from 'axios';
 import PersonalMenu from '../PersonalMenu';
-import { Grid, Breadcrumbs, Typography } from '@material-ui/core';
+import { Grid, Breadcrumbs, Typography, Chip } from '@material-ui/core';
 import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  subtitle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+});
 
 function ResourcePage({ matchProps }) {
   const [resource, setResource] = useState({});
+  const [loading, setLoading] = useState(true);
 
   useEffect(() => {
     axios
       .get('/api/v1/resources/' + matchProps.match.params.guid)
       .then(function(response) {
         // handle success
         setResource(response.data);
+        setLoading(false);
       })
       .catch(function(error) {
         // handle error
         console.log(error);
       });
-  }, [matchProps.match.params.id]);
+  }, [matchProps.match.params.guid]);
+
+  const classes = useStyles();
+
+  const {
+    title,
+    url,
+    author,
+    tags,
+    media_type: mediaType,
```

Comment:
@bengineerdavis - can you explain what's going on here? Is this what you were mentioning about converting Python to JavaScript? I'm not sure I fully understood what was going there.

Reply:
Sure: we were getting ESLinting errors for `media_type` and `date_published` because they aren't valid JavaScript variables, so our solution was to alias those keys coming back from the JSON blob so it wouldn't be an issue for React when the frontend code was executed.

Reply:
Thanks to @angelocordon this got resolved so we don't need the aliasing anymore.